src/app/modules/verification/__tests__/verification.model.spec.ts

```diff
@@ -535,7 +539,7 @@ describe('Verification Model', () => {
       })
 
       it('should return null when the transaction ID is not found', async () => {
-        const result = await VerificationModel.updateHashForField({
+        const result = await VerificationModel.updateHashForFormField({
```

Comment:
Did we add tests for the payment versions of these functions?

Reply:
Oh... I didn't :') thanks for spotting this! will add them

Reply:
Holy, after I added the tests for the refactor (including those related to the above comment), the lines of code got bumped up to 3k (from 1.9k) :')